IdentityCore/src/controllers/MSIDLocalInteractiveController.m

```diff
@@ -78,6 +79,16 @@ - (void)acquireToken:(MSIDRequestCompletionBlock)completionBlock
 
     MSIDRequestCompletionBlock completionBlockWrapper = ^(MSIDTokenResult * _Nullable result, NSError * _Nullable error)
     {
+        
+        if (result && [result isKindOfClass:MSIDSSONonceRedirectResponse.class])
```

Comment:
Not sure if it's the best place to add this. I'd prefer local controller to not have to know about SSO nonce redirects. Seems too low level for the controller. Any other way to implement this?

Reply:
Removed parsing sso nonce response from here. Added a condition to not log error if it contains sso nonce over here though, please let me know if it is not necessary.